packages/story-editor/src/components/mediaRecording/footer.js

```diff
@@ -244,17 +231,20 @@ function Footer({ captureImage, videoRef }) {
       args.additionalData.isMuted = isMuted;
       args.additionalData.isGif = isGif;
 
-      // Get length from the video blob on screen and fall back
-      // to the duration in state if needed since it's not super reliable.
-      const { length, lengthFormatted } = getVideoLength(videoRef.current);
-      args.resource.length = length || duration;
-      args.resource.lengthFormatted = length
-        ? lengthFormatted
-        : getVideoLengthDisplay(duration);
+      args.resource.length = duration;
```

Comment:
@barklund This originally used `getVideoLength(videoRef.current)` whenever possible because it was more accurate. Is that not possible anymore?

Reply:
Only when there's no trim data. When there is, the video element length doesn't actually reflect the final video length (the trim offsets do). But I can revert so it will use the former if there's no trim.